static/app/utils/metrics/virtualMetricsContext.tsx

```diff
@@ -61,304 +52,7 @@ const Context = createContext<ContextType>({
   isLoading: false,
 });
 
+// TODO: Remove this when all dependet code was cleaned up
 export function useVirtualMetricsContext() {
```

Comment:
There is still a lot of code accessing the context.
Though this will simply disable all span metric features, if the provider is not available.